### DISCUSSION TITLE
Oops! Could not connect to this Shlink server.

### DISCUSSION THREAD

**Discussion by franco640** [shlinkio/discussions#16] [Help wanted]
### shlink-web-client version

shlink-web-client_4.1.0_dist

### How do you use shlink-web-client

Self-hosted

### Summary

I have finally got the backend working but now want the webclient to work 

I used the following 

# download shlinkio web client
sudo wget https://github.com/shlinkio/shlink-web-client/releases/download/v4.1.0/shlink-web-client_4.1.0_dist.zip
# extract the zip archive
sudo unzip shlink-web-client_4.1.0_dist.zip
# move the extracted files to the web root
sudo mv ./shlink-web-client*/* /var/www/html
sudo nano /etc/apache2/sites-available/shlinkio.conf

Added the shlinko.conf

Alias /shlinkio "/var/www/shlinkio/public/"
<Directory /var/www/shlinkio/public>
Options FollowSymlinks Includes ExecCGI
AllowOverride All
Require all granted
Order allow,deny
allow from all
</Directory>

enable the shlinkio site
sudo a2ensite shlinkio
# restart apache2 service for the changes to take effect
sudo systemctl restart apache2
# generate an API key
sudo -u www-data php /var/www/shlinkio/bin/cli api-key:generate

I then get 

Oops! Could not connect to this Shlink server.

Make sure you have internet connection, and the server is properly configured and on-line.

Can i ask what I have done wrong here 

thanks

**Comment by acelaya** [comment]
You should be able to see the actual error in the browser's console. Once you have that, there are common issues explained here https://shlink.io/documentation/troubleshooting/

**Reply by franco640** [reply]
Failed to load resource: the server responded with a status of 404 (Not Found) 

Is this to do with permissions again? does it need to be created before hand as that's what get unzipped and placed in the 
sudo mv ./shlink-web-client*/* /var/www/html


`drwxr-xr-x 2 root root  4096 Mar 17 11:27 assets
-rw-r--r-- 1 root root   642 Mar 17 11:27 favicon.gif
-rw-r--r-- 1 root root  1165 Mar 17 11:27 favicon.ico
-rw-r--r-- 1 root root  1165 Mar 17 11:27 favicon.png
-rw-r--r-- 1 root root  2248 Mar 17 11:27 favicon.svg
drwxr-xr-x 2 root root  4096 Mar 17 11:27 icons
-rw-r--r-- 1 root root  6100 Mar 17 11:27 index.html
-rw-r--r-- 1 root root   612 Mar 25 14:24 index.nginx-debian.html
-rw-r--r-- 1 root root  2074 Mar 17 11:27 manifest.json
-rw-r--r-- 1 root root 28477 Mar 17 11:27 service-worker.js`

When i then enter the API key 

index-e7ikdlm2.js:851 GET http://shortner.uta.lk//rest/v3/health 404 (Not Found)

**Reply by acelaya** [reply]
This is a misconfiguration in your Shlink instance's Apache, not a problem with shlink-web-client

**Reply by acelaya** [reply]
On a second check, I just noticed http://shortner.uta.lk resolves a shlink-web-client instance, so you are setting the wrong URL for the Shlink server.
